catalog-data.ttl

```diff
@@ -7,10 +7,10 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <https://elf-pavlik.hackers4peace.net> a ex:Person ;
-  ex:description "One of Solid CG co-chairs. Co-editor of Solid Application Interoperability, Solid-OIDC, Security Consi  derations and some Solid Notification channel types. Maintainer of sai-js."@en ;
+  ex:description "One of Solid CG co-chairs. Co-editor of Solid Application Interoperability, Solid-OIDC, Security Consi  derations and some Solid Notification channel types. Maintainer of sai-js." ;
   ex:landingPage <https://elf-pavlik.hackers4peace.net> ;
   ex:logo <https://avatars.githubusercontent.com/u/876431?v=4> ;
-  ex:name "elf Pavlik"@en ;
```

Comment:
@jeff-zucker do you know how did my name get `@en` tag?
Nothing important just wondering where it was assigned this way. Does your form add set it to any text input?